index.html

```diff
@@ -264,15 +257,11 @@ <h1 data-l10n-id="save-as"></h1>
 			<button data-click="navBack" data-l10n-title="close"><span class="icon icon-close"></span></button>
 			<h1 data-l10n-id="open"></h1>
 		</header>
-		<div role="main" class="header-only">
+		<div role="main" class="header-only" id="open-dialog-main-area">
 			<article data-type="list" class="docsList">
-				<div id="open-dialog-device-area">
-					<h2 data-l10n-id="documents-device"></h2>
-					<ul id="open-dialog-device-list"></ul>
-				</div>
-				<div id="open-dialog-dropbox-area">
-					<h2 data-l10n-id="documents-dropbox"></h2>
-					<ul id="open-dialog-dropbox-list"></ul>
+				<div id="open-dialog-recents-area">
+					<h2 data-l10n-id="recent-documents"></h2>
```

Comment:
That is a good idea. Maybe, it could say "Sort By: <Dropdown>".

We could also add a search box.